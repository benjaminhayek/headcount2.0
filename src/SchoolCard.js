import React from 'react';

const SchoolCard = ({location, data}) => {
return (
  <div>
    <h1>{Object.keys(data)}</h1>
    <ul>
      {Object.keys(data).map(location=> Object.keys(data[location]).map(year => <li>{location}: {data[location][year]}</li>))}
    </ul>
  </div>
  )
}

export default SchoolCard;