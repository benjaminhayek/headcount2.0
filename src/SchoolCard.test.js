import React from 'react';
import ReactDOM from 'react-dom';
import SchoolCard from './SchoolCard';
import { shallow, mount} from 'enzyme';

describe('IdeaForm', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<SearchForm searchSchool={searchSchoolMock} />)
  })

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })

})