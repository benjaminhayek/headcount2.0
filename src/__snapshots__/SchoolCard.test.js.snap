// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IdeaForm matches the snapshot 1`] = `undefined`;
